metadata.json

```diff
@@ -8,7 +8,10 @@
   "project_page": "https://github.com/rnelson0/puppet-certs",
   "issues_url": "https://github.com/rnelson0/puppet-certs/issues",
   "dependencies": [
-
+    {
```

Comment:
Looks like this got left in, probably because I haven't figured out how to do multi-line suggestions.